### PR TITLE
Skeleton

### PR DESCRIPTION
**Pull request by waitinfuture**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.aliyun</groupId>
+  <artifactId>remote-shuffle-service</artifactId>
+  <version>0.3.0-SNAPSHOT</version>
```
Comment (TanYuxin-tyx) [inline pom.xml:26]: Why did we start the version from `0.3.0-SNAPSHOT`?

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,574 @@
...
+        <groupId>org.slf4j</groupId>
+        <artifactId>jcl-over-slf4j</artifactId>
+        <version>${slf4j.version}</version>
+        <!-- runtime scope is appropriate, but causes SBT build problems -->
+      </dependency>
+      <dependency>
+        <groupId>junit</groupId>
+        <artifactId>junit</artifactId>
+        <scope>test</scope>
+        <version>4.12</version>
```
Comment (TanYuxin-tyx) [inline pom.xml:124]: I think we'd better use `<properties>` to manage all the dependency versions in this project, which is clearer

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,574 @@
...
+      </dependency>
+      <dependency>
+        <groupId>org.scalatest</groupId>
+        <artifactId>scalatest_${scala.binary.version}</artifactId>
+        <scope>test</scope>
+        <version>3.2.3</version>
```
Comment (TanYuxin-tyx) [inline pom.xml:130]: Ditto, and other versions in this or other `pom.xml` should also be managed in `<properties>`

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,574 @@
...
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </pluginRepository>
+  </pluginRepositories>
+
+  <dependencyManagement>
+    <dependencies>
```
Comment (TanYuxin-tyx) [inline pom.xml:93]: Currently, we have listed all the dependencies in advance, why not add them before using?

File: pom.xml (hunk 1 above)
Comment (waitinfuture) [reply pom.xml:26]: We have released 0.1.0 and the master is 0.2.0-SNAPSHOT. I think we can make it 1.0.0-SNAPSHOT.

File: pom.xml (hunk 4 above)
Comment (waitinfuture) [reply pom.xml:104]: Versions and exclusions have been carefully tuned, so I think it's better to inherit.

**Review by TanYuxin-tyx — APPROVED** [review]

**Pull request merged**